.github/workflows/coverage_runner.yaml

```diff
@@ -39,9 +39,9 @@ jobs:
           go-version: "${{ matrix.go_version }}"
 
       - name: Install JDK      
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v4
         with:
-          distribution: 'adopt'
+          distribution: 'temurin'
           java-version: '8'
```

Comment:
What about java version? Maybe we can update it?

Reply:
> What about java version? Maybe we can update it?

I think we _can_ upgrade it, but I'm not sure we need to. The [documentation says that Java 8 is ok for the integration tests](https://github.com/hazelcast/hazelcast-go-client?tab=readme-ov-file#running-the-tests).

Reply:
I think it would be better to set it to the minimal version required by Hazelcast.

Reply:
> I think it would be better to set it to the minimal version required by Hazelcast.

e09a31ecdf176fee7d7363c4ebb894fd217bab64